### PR TITLE
config changes to player and team modules, team also has simple sort …

### PR DESCRIPTION
**Pull request by RBecker262**
Both the team and player update modules got the new config file logic to get the data directory.

Also cleaned up Player module for earned runs, used int() instead of float() to get earned runs from dictionary since there's no such thing as half a run. - 1 line change.

Team module also had bug due to random loading of dictionary in 3.4 (changed in 3.5 to stay sorted), so had to sort dictionary keys before staring loop as when processing a double header, game 2 must update the team after after game 1. - also a 1 line change.

### CODE REVIEW

**Review by alesage — CHANGES_REQUESTED** [review]
Pretty sure this repeats some material from another PR, else it'd be two lines :) , need to get into this branch and merge master.

**Comment by RBecker262** [comment]
Not sure what you mean that this repeats from another PR. I deleted (closed) the PR I created late Friday night and split them into 3 you saw this morning. None of the various modules were duplicated in any of the three branches. You're seeing a lot of the same code because it is a lot of the same code. Adding the Config file for to retrieve the Data directory pretty much has the same look across updplayer.py and updteam.py in this PR, and in dailysched.py in one of the other PR's. So yeah you've seen this code three times now but in 3 different modules...two of the modules in this PR and one module in another PR.

**Comment by RBecker262** [comment]
Double checked again to verify, this is what was in each PR

#16 - sample backend config, cron.yaml, and requirements.txt

#17 - masterbuild.py and dailysched.py

#18 - updteam.py and updplayer.py

File: backend/updplayer.py [hunk 1]
```diff
@@ -64,6 +74,35 @@ def get_command_arguments():
     return argue
 
 
+def get_data_directory(config_file):
```
Comment (alesage) [inline backend/updplayer.py]: This ftn, e.g., is also in #17 and I think it belongs there and not here.

**Comment by RBecker262** [comment]
I was replying to your comment on the function and it was deleted.  I had written this...

Correct. It exists there in #17 as the module there (dailysched.py) needs the directory information to know where to write its output file. This module updteam.py needs to know where to write the team master file, and where the input schedule file is for that matter so this module also has to retrieve the data directory from the config file.

Also, the updteam.py module included in this pull has the same function too but you did not mention that one being repeated.

**Comment by alesage** [comment]
I see now, I confess that that duplication faked me out.  New criticism would be to just move those utility ftns to a utils.py module or some such and import them as they appear to be identical.

**Comment by RBecker262** [comment]
Agreed. I started to wonder if that was the point you were trying to get to and didn't think of it myself until I made my last comment. Want me to change these two before merging? You already merged the other so you can open an issue to fix that one at some point.

**Comment by alesage** [comment]
Good question, yes please, let's pretend this commit really is coming later in time and the inspiration to make a utils.py or some such to consolidate config stuff coincides with this new functionality.

**Comment by RBecker262** [comment]
And there you have it, a new function called myutils.py and I made it generic so it will work for any config file, any section, and any key. Fully tested that it works as well as handles the error conditions coded. Both the updteam and updplayer modules were changed to reference this so this might look bigger with a third piece now, but they all go together and the changes in the two master update modules are smaller since they no longer have the function/class definitions. My comment is probably longer. :)

**Review by alesage — APPROVED** [review]
Ok thanks for that change.  Onward!

**Pull request merged**